.github/workflows/builds.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.9', '3.10']
+        python-version: ['3.9', '3.10']
```

Comment:
why removing support for python 3.8?

Reply:
Because readthedocs doesn't know how to deal with one of the typing features (I am blanking on which one, but one of the things I introduced in the mm.typing module) unless python is >= 3.9. It was throwing an error and this was the only solution.